src/data/clients.ts

```diff
@@ -58,6 +58,15 @@ export const clients: ClientData[] = [
     milestone: 0,
     contact: [],
   },
+  {
+    description: "JAM client implementation in Rust",
+    homepage: "https://universaldot.foundation",
+    name: "UNIVERSALDOT",
+    lang: "Rust",
+    lang_set: "B",
+    milestone: 1,
```

Comment:
Probably milestone 0 instead of 1. It's indicating the finished milestones. Key is not very informative about that. 👍 
```suggestion
    milestone: 0,
```